packages/cli/package.json

```diff
@@ -9,7 +9,7 @@
     "chalk": "^5.3.0",
     "yargs": "^17.7.2",
     "@quality-metrics/models": "^0.0.1",
-    "@quality-metrics/utils": "^0.0.1",
-    "zod": "^3.22.1"
+    "zod": "^3.22.1",
+    "@quality-metrics-cli/core": "*"
```

Comment:
We've been using `^0.0.1` for our package deps, not `*`. I guess restricting it to major versions is safer, but either way we should be consistent.